dpnp/backend/include/dpnp_iface.hpp

```diff
@@ -124,7 +124,7 @@ INP_DLLEXPORT void dpnp_all_c(void* array, void* result, const size_t size);
  * @param [out] result      Output array.
  * @param [in]  size        Number of input elements in `array`.
  */
-template <typename _DataType, typename _ResultType>
+template <typename _DataType>
```

Comment:
Leave it as it was. See, https://github.com/IntelPython/dpnp/pull/637#discussion_r586731916